data/built-ins.json

```diff
@@ -100,26 +100,31 @@
   },
   "es6.map": {
     "chrome": 51,
+    "firefox": 53,
```

Comment:
Not sure that iterator closing is such significant feature for Map and other new types. FF can deal with these types since 30/40th versions